### PR TITLE
show news detail

### PR DESCRIPTION
**Pull request by at-dieulexuan**

### CODE REVIEW

File: config/constants.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+	return [
+		'LIMIT_NEWS_INDEX' => '2',
```
Comment (ghost) [inline config/constants.php:3]: no need this `const` the Eloquent have the `perPage` property, default is 15, If you want to custom, just define in the model class

File: resources/lang/en/footer.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
```
Comment (ghost) [inline resources/lang/en/footer.php:1]: why this file is in `en` folder, but the content is VietNamese

File: resources/views/danabus/news/news.blade.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <div id="single-project text-news">       
+            {{$news->content}}
+        </div > 
+      </div>
+    </div>  
+    <div id="contact" class="contact">
+      <div class="section secondary-section">
+        <div class="container">
+          <div class="title">
+              <h2 class="map">{{ trans('index.map')}}</h2>
```
Comment (ghost) [inline resources/views/danabus/news/news.blade.php]: I don't see the lang `index.map`

**Comment by ghost** [comment]
@at-dieule please commit controller also

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Danabus/NewsController.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+{
+    /**
+     * Display a list news
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $news = News::paginate();
+        $cats = Category::all();
```
Comment (ghost) [inline app/Http/Controllers/Danabus/NewsController.php:20]: Why you need `$categories` in this request?

File: app/Http/Controllers/Danabus/NewsController.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Display the news detail
+     *
+     * @param int $id id of news
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $news = News::find($id);
```
Comment (ghost) [inline app/Http/Controllers/Danabus/NewsController.php]: `findOrFail` is better

File: config/constants.php (hunk 1 above)
Comment (ghost) [inline config/constants.php:3]: pls remove

**Review by ghost — CHANGES_REQUESTED** [review]
Pls check my cmt

File: app/Http/Controllers/Danabus/NewsController.php (hunk 4 above)
Comment (at-dieulexuan) [reply app/Http/Controllers/Danabus/NewsController.php:20]: because on index page i will show categories and filter news

**Pull request closed without merging**